woocommerce/woocommerce
# fix style for tracks prop `suggestion_slug` (lower_case not camelCase)

### Changes proposed in this Pull Request:
Corrects the style of the tracks property `suggestion_slug` (was `suggestionSlug`) when sending Marketplace Suggestions tracks events.

### How to test the changes in this Pull Request:
1. View a page with marketplace suggestions (e.g. products, product edit, orders empty)
2. View, click or, dismiss suggestion(s).
3. See appropriate and valid tracks events getting sent.
